q2_quality_control/_threshold_graph/_visualizer.py

```diff
@@ -40,16 +41,31 @@ def _check_inputs(**kwargs):
 
 TEMPLATES = pkg_resources.resource_filename(
     'q2_quality_control._threshold_graph', 'assets')
+_blast_url_template = ("http://www.ncbi.nlm.nih.gov/BLAST/Blast.cgi?"
+                       "ALIGNMENT_VIEW=Pairwise&PROGRAM=blastn&DATABASE"
+                       "=nt&CMD=Put&QUERY=%s")
 
 
-def decontam_score_viz(output_dir, decontam_scores: qiime2.Metadata,
-                       table: pd.DataFrame, threshold: float = 0.1,
+def format_fasta(header, sequence):
```

Comment:
Don't reimplement this, as you're going to need to write unit tests for it, handle edge cases, etc. 

I think a better way to do this is to bring your `rep_seqs` in as a DNAInterator (see an example [here](https://github.com/caporaso-lab/q2-dwq2/pull/12/files#diff-12dca64616ef2e38c0f6ad05c6050f649209daab810693d78a3ece281d0effbe)). Then, you can iterate over the items to write them to file - for example:

```python
with open('test.fasta', 'w') as fh:
    for seq in rep_seqs:
        seq.write(fh)
```

Reply:
This was removed in favor of the data being read in as a DNAIterator as suggested

addressed in [adfe12a](https://github.com/qiime2/q2-quality-control/pull/82/commits/adfe12ad4213c9923face4267028d18902427b8a)